README.md

```diff
@@ -2102,6 +2103,53 @@ The error message thrown by mailgun is passed to the response object when an ema
 Mailgun error codes can be found
 [here](https://documentation.mailgun.com/en/latest/api-sending.html#error-codes).
 
+## Localization
+
+### Setting application properties
+
+Form flow library will read the languages set in an apps `application.properties`.
```

Comment:
A little nit pick:  There's a few spots where you say "app" or "apps", I think it might read easier if the word "application" was used.